src/configure/configure.ts

```diff
@@ -512,13 +514,13 @@ class Orchestrator {
             appropriatePipelines = remotePipelines;
         }
         const pipelineMap = this.getMapOfUniqueLabels(appropriatePipelines);
+        this.filterPipelinesByResourceType(pipelineMap);
```

Comment:
What should happen if dockerfile does not exist? I think we should show an error to user and tell them to add a dockerfile and try again. This should happen immediately after repo analysis.

Reply:
As of now, if there isn't any dockerFil, this would mean that the template service will not return AKS template and after the filterByResource, the templates to proceed will be 0 which will give the error that the language of the repo is not supported.

Reply:
So, based on your [comment](https://github.com/microsoft/vscode-deploy-azure/pull/165/files#r494124525), this comment will change to: what should happen if returned list does contain any template with target of AKS?

Reply:
So, the templates with AKS target are filtered. If they are more than one, user gets to choose, otherwise, considering present scenario, if there is one, we proceed as it is. This is in sync with the existing behavior that we have.